.github/workflows/ci.yml

```diff
@@ -41,6 +41,19 @@ jobs:
           name: logs
           path: test/server.log
 
+  index_resources:
+    if: |2
+         github.repository == 'nextstrain/nextstrain.org'
+      && github.event_name == 'push'
+      && github.ref == 'refs/heads/master'
+    needs: test
+    permissions:
+      id-token: write # needed to interact with GitHub's OIDC Token endpoint
+      contents: read
+    uses: ./.github/workflows/index-resources.yml
+    with:
+      ref: ${{ github.sha }}
```

Comment:
_follow-up to prior comment_

I think `github.sha` will be the same inside the called workflow without having to pass it thru like this.

Reply:
Thanks for talking through this in https://github.com/nextstrain/nextstrain.org/pull/841#discussion_r1586923809. Now the workflow just directly references `GITHUB_SHA`.